src/CalcManager/Ratpack/conv.cpp

```diff
@@ -49,7 +49,7 @@ wchar_t g_decimalSeparator = L'.';
 #define Calc_UInt32x32To64(a, b) ((uint64_t)((uint32_t)(a)) * (uint64_t)((uint32_t)(b)))
 #endif
 
-#elif defined(_M_IX86) || defined(__i386__) || defined(_M_ARM) || defined(__EMSCRIPTEN__)
+#elif defined(_M_IX86) || defined(__i386__) || defined(_M_ARM) || defined(__EMSCRIPTEN__) || defined(__APPLE__)
```

Comment:
This may not be the right one, as it's targeting x86, and for iOS it's using ARM64.

Reply:
fixed